doc/source/overview/config_files.md

```diff
@@ -0,0 +1,753 @@
+  -
+* - sigma
+  - List of floats to pass as bandwidths to kernel. Only used if 
+  [GaussianRBF](https://docs.seldon.io/projects/alibi-detect/en/latest/api/alibi_detect.utils.tensorflow.html) specified.
+  - `None`
+* - trainable
+  - `True`/`False`. Whether or not to track gradients w.r.t. sigma, allowing the kernel to be trained. Only used if 
+  [GaussianRBF](https://docs.seldon.io/projects/alibi-detect/en/latest/api/alibi_detect.utils.tensorflow.html) specified. 
+  - `False`
+* - kwargs
+  - Dictionary of additional keyword arguments to pass to the kernel. Only used if kernels other than 
+  [GaussianRBF](https://docs.seldon.io/projects/alibi-detect/en/latest/api/alibi_detect.utils.tensorflow.html) specified.  
+  - `{}`
+```
+
+The default kernel for [MMDDrift](../api/alibi_detect.cd.mmd.rst) and other detectors is the 
+[GaussianRBF](https://docs.seldon.io/projects/alibi-detect/en/latest/api/alibi_detect.utils.tensorflow.html) kernel.
+This Alibi Detect class is pre-registered (see [Registering artefacts](registering_artefacts)), meaning it can be 
+specified with `src = "@utils.tensorflow.kernels.GaussianRBF"` (replace `tensorflow` with `pytorch` for the pytorch
+version). If the specified kernel is a `GaussianRBF` kernel, the `sigma` and `trainable` kwargs are passed to it. For all other 
+kernels, the generic `kwargs` dict is passed.
+
+The [LearnedKernel](https://docs.seldon.io/projects/alibi-detect/en/latest/cd/methods/learnedkerneldrift.html) detector
+requires a [DeepKernel](https://docs.seldon.io/projects/alibi-detect/en/latest/api/alibi_detect.utils.tensorflow.html?highlight=deepkernel#alibi_detect.utils.tensorflow.DeepKernel)
+to be passed. This is also specified by the `kernel` field, but with a slightly different schema: 
+
+```{list-table} DeepKernel kernel schema
+:header-rows: 1
+:name: deepkernel-table
+:widths: auto
+
+* - Field
+  - Description
+  - Default (if optional)
+* - `kernel_a`
+  -  Kernel to apply to projected inputs. A string referencing a kernel serialized in a `.dill` file, a registered 
+  kernel object, or a kernel artefact dictionary (see {ref}`kernel-table`).
+  - `"@utils.tensorflow.kernels.GaussianRBF"` <br/>(with `trainable = True`)
+* - `kernel_b`
+  -  Kernel to apply to raw inputs. A string referencing a kernel serialized in a `.dill` file, a registered kernel 
+  object, or a kernel artefact dictionary (see {ref}`kernel-table`).
+  - `None`
+* - `proj`
+  - Projection to be applied to the inputs. Should be a TensorFlow model specified as a registered 
+  model, or as an artefact dictionary following the layout in {ref}`model-table`. 
+  - 
+* - `eps`
+  - The proportion (in [0,1]) of weight to assign to `kernel_b`. Specified as a float, or set to `"trainable"`. 
+  - `"trainable"`
+```
+
+As shown in the example below, when a `DeepKernel` is specified, `kernel_a` and `kernel_b` are themselves kernels, 
+and can be specified via the {ref}`kernel-table`.
```

Comment:
It's not clear to me from the schema that a `DeepKernel` is specified, it seems implicit due to the presence of `kernel_a` and `kernel_b`?